WebHostLib/static/assets/tutorial/ff1/multiworld_en.md

```diff
@@ -46,69 +39,30 @@ you prefer, or it is your first time playing select the "Archipelago" preset on
 It should download two files. One is the `*.nes` file which your emulator will run and the other is the yaml file
 required by Archipelago.gg
```

Comment:
Right here is where I would like the link the the main tutorial which just points out that you need to generate and host a game before you can start connecting. Essentially just state that the workflow here is to wait until the game is hosted. This is just an example for the rough concept, feel free to reword.

`At this point you are ready to join the multiworld. If you are uncertain on how to generate, host or join a multiworld please refer to the game agnostic guide[link]`